routing.rst

```diff
@@ -1432,8 +1508,8 @@ you decided not to maintain it anymore), you can deprecate its definition:
             )
         ;
 
-In this example, every time the ``new_route_name`` alias is used, a deprecation
-warning is triggered, advising you to stop using that alias.
+In this example, every time the ``some_route_name`` alias is used, a deprecation
+warning is triggered, advising you to stop using this route and prefer using ``new_route_name``.
```

Comment:
We could update the deprecation messages in examples to encourage using the new route.